res/database/Default/user/therob.xml

```diff
@@ -1727,54 +1955,64 @@
 					<title>
 						<de>Byte beißt den Wärter</de>
 						<en>Byte bites the guardian</en>
+						<pl>Byte gryzie strażnika</pl>
 					</title>
 					<description>
 						<de>Aus der bewachten Gruft entkommen. Doch der Wärter stört.</de>
 						<en>Escape from the guarded crypt. But the warden is interfering.</en>
+						<pl>Ucieczka ze strzeżonej krypty. Ale strażnik próbuje przeszkodzić w tym.</pl>
 					</description>
 					<ratings critics="13" speed="59" />
 				</programme>
 				<programme id="therob-programme-ser-RonnyunddasBytekehrenzurueck-ep02" product="5" fictional="1" licence_type="2" tmdb_id="0" imdb_id="" creator="8751" created_by="TheRob" comment="Reference: https://de.wikipedia.org/wiki/Robbi,_Tobbi_und_das_Fliewat%C3%BC%C3%BCt">
 					<title>
 						<de>[0|First] sucht das Fliwatüt</de>
 						<en>[0|First] is looking for the Flywatoot</en>
+						<pl>[0|First] szuka Muchołówki</pl>
 					</title>
 					<description>
 						<de>[0|First] jagt das Fliwatüt. Die Armee der Omas stellt sich ihm entgegen.</de>
 						<en>[0|First] is chasing the Flywatoot. The army of grannies opposes him.</en>
+						<pl>[0|First] szuka Muchołówki. Przeciwstawia mu się armia babć.</pl>
```

Comment:
"Muchołówki" seems to be a "real" word. Fliwatüt / Flywatoot is an artificial word (but from a real 60s German childrens book) consisting of flying - water - tooting. We could even include the TV series eventually. https://www.imdb.com/title/tt0358872/releaseinfo/ — maybe there is a Polish version after all?

Reply:
https://www.filmweb.pl/film/M%C3%B3j+przyjaciel+robot-2016-784585

Reply:
https://www.filmweb.pl/serial/Robbi%2C+Tobbi+und+das+Fliewat%C3%BC%C3%BCt-1972-370096

Unfortunately there is no translation for this.

adapted.